src/plugins/data/server/server.api.md

```diff
@@ -22,6 +22,7 @@ import { CatTasksParams } from 'elasticsearch';
 import { CatThreadPoolParams } from 'elasticsearch';
 import { ClearScrollParams } from 'elasticsearch';
 import { Client } from 'elasticsearch';
+import { ClientOptions } from '@elastic/elasticsearch';
```

Comment:
no idea why it's here. it's not used in the file

Reply:
🤷‍♂️